src/Tests/IQsharpEngineTests.cs

```diff
@@ -470,7 +470,7 @@ public async Task TestWho()
             var result = response.Output as string[];
             PrintResult(response, channel);
             Assert.AreEqual(ExecuteStatus.Ok, response.Status);
-            Assert.AreEqual(5, result?.Length);
+            Assert.AreEqual(6, result?.Length);
```

Comment:
Not sure I understand why this is 6 now. Did the `HelloQ` example change?

Reply:
This assertion checks how many operations are defined in the test snippets. Since the new test uses a new operation, this needs to be incremented to prevent a failure when testing `%who`.